code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -217,8 +217,17 @@
 	if(!istype(M))
 		return
 	if(isoozeling(M))
-		M.blood_volume -= 30
-		to_chat(M, "<span class='warning'>The water causes you to melt away!</span>")
+		if (method == TOUCH)
+			M.adjustStaminaLoss(25, 0)
+			to_chat(M, "<span class='warning'>You struggle to maintain your form!</span>")
```

Comment:
This should have some other effect I think maybe a second of knockdown with eyes being blurred, they are weak to water since they have a huge, huge buff of not being able to be set on fire in the first place.